drivers/led/ht16k33.c

```diff
@@ -123,7 +118,7 @@ static int ht16k33_led_blink(const struct device *dev, uint32_t led,
 		cmd |= HT16K33_OPT_BLINK_2HZ;
 	}
 
-	if (i2c_write(data->i2c, &cmd, 1, config->i2c_addr)) {
+	if (i2c_write(config->i2c_dev, &cmd, 1, config->i2c_addr)) {
```

Comment:
instead of hardcoded size 1, please use sizeof()

Reply:
Fixed.